pkg/debug/spec.sh

```diff
@@ -530,6 +530,30 @@ __EOT__
      COMMA="},"
   fi
 done
+#enumerate physical CAN devices
+for CAN in /sys/class/net/*; do
+    TYPE=$(cat "${CAN}/type")
+    if [ "$TYPE" != 280 ]; then
+        continue
+    fi
+    ZTYPE="IO_TYPE_CAN"
+    IFNAME=$(basename "${CAN}")
+    cat <<__EOT__
+    ${COMMA}
+    {
+     "ztype": ${ZTYPE},
+     "phylabel": "${IFNAME}",
+     "phyaddrs": {
+       "Ifname": "${IFNAME}"
```

Comment:
Can CAN device have PCI (or USB) address? Should we include them here as well?

Reply:
depending on the controller Yes, but QEMU just use the ifname to do the "passt-through", which as explained in the docs, is not a real pass-through, but an emulation.... I have nothing to do with PCI and USB addresses in this case, so that's why they are omitted. For ARM devices usually the controllers are implemented in the SoC, so they don't have PCI or USB address.